resources/quickstart/waas/web-quickstart.mdx

```diff
@@ -95,72 +89,51 @@ To retrieve these values for configuration within your application, follow these
-
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <AuthCoreContextProvider
-      options={{
-        projectId: process.env.REACT_APP_PROJECT_ID,
-        clientKey: process.env.REACT_APP_CLIENT_KEY,
-        appId: process.env.REACT_APP_APP_ID,
-        wallet: {
-          visible: true,
-        },
-      }}
-    >
-      <App />
-    </AuthCoreContextProvider>
-  </React.StrictMode>
-)
-```
+import { InteractiveLayout } from '/snippets/InteractiveLayout.mdx';
+
+<InteractiveLayout />
 
 ***
 
 ## Facilitating logins
 
-With the SDK configured, you can move on to initiating social logins within your application. Once again, this will deviate between Particle Auth and Particle Auth Core, but the general idea remains the same: If you're using Particle Auth, social login can be requested through the same instance of `ParticleNetwork` that was recently configured, such as with:
+With the SDK configured, you can move on to initiating social logins within your application. Particle Auth contains [numerous relevant hooks](https://github.com/Particle-Network/particle-auth-ethers-demo/blob/7cf947b6c8a05731f285821b03134a6317e0e53f/particle-next-starter/src/app/page.tsx#L23), although we'll specifically be focusing on `useConnect`, which contains a function, `connect`, that directly facilitates [wallet creation]() through social login.
 
-```javascript JavaScript
-// authType can be 'google', 'twitter', 'email', 'github', 'facebook', etc.
-await particle.auth.login({authType}); 
-```
-
-If you're using Particle Auth Core, you can use `connect` from the `useConnect` hook imported from `@particle-network/auth-core-modal`. Doing so looks like this:
+Upon logging in, the embedded wallet modal included with Particle Auth will be accessible through the button in the bottom right, unless otherwise specified through `wallet` within `AuthCoreContextProvider`.
 
 ```javascript JavaScript
 import { useConnect } from '@particle-network/auth-core-modal';
-import { X1Testnet } from '@particle-network/chains';
+import { Base } from '@particle-network/chains';
 
 const { connect } = useConnect();
 
 await connect({
-    // authType can be 'google', 'twitter', 'email', 'github', 'facebook', etc.
-	socialType: authType, 
-	chain: X1Testnet,
+	chain: Base
 });
 ```
 
+<Tip>The complete Web SDK reference is available at [Web (JavaScript/TypeScript)](/api-reference/auth/desktop-sdks/web). Additionally, a demo repository for Next.js can be found [here](https://github.com/Particle-Network/particle-auth-ethers-demo/tree/main/particle-next-starter), alongside a demo for create-react-app [here](https://github.com/Particle-Network/particle-auth-ethers-demo/tree/main/particle-auth-cra).</Tip>
+
 ***
 
-<Tip>The complete Web SDK reference is available at [Web (JavaScript/TypeScript)](/api-reference/auth/desktop-sdks/web).</Tip>
\ No newline at end of file
+### Related Resources
```

Comment:
Similar degrees of consistent cross-linking resources needs to be replicated across pages; should be as simple as possible for developers to find what they want (such as customization).